### PR TITLE
FFA-7 setup detekt

### PR DESCRIPTION
**Pull request by Coshiloco**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -84,3 +85,31 @@ dependencies {
...
+        html.required.set(true) // observe findings in your browser with structure and code snippets
+        xml.required.set(true) // checkstyle like format mainly for integrations like Jenkins
+        txt.required.set(true) // similar to the console output, contains issue signature to manually edit baseline files
+        sarif.required.set(true) // standardized SARIF format (https://sarifweb.azurewebsites.net/) to support integrations with GitHub Code Scanning
+        md.required.set(true) // simple Markdown format
+    }
+}
+
+tasks.withType(io.gitlab.arturbosch.detekt.Detekt).configureEach {
+    jvmTarget = "1.8"
```
Comment (foliolo) [inline app/build.gradle:110]: Can this be changed to java 17?

File: app/build.gradle [hunk 2]
```diff
@@ -84,3 +85,31 @@ dependencies {
...
+}
+
+tasks.withType(io.gitlab.arturbosch.detekt.DetektCreateBaselineTask).configureEach {
+    jvmTarget = "1.8"
```
Comment (foliolo) [inline app/build.gradle:114]: This one too

**Review by foliolo — CHANGES_REQUESTED** [review]
There isn't any config file?

**Review by foliolo — APPROVED** [review]
Looks good after all!

**Review by hgarciaalberto — COMMENTED** [review]
Looks good

**Pull request merged**